CONTRIBUTING-CODE.adoc

```diff
@@ -77,6 +80,29 @@ For instance to build against Asciidoctor (Ruby) v2.0.10 use:
 
  $ ASCIIDOCTOR_CORE_VERSION=v2.0.10 npm run build --prefix packages/core
 
+==== Building against a local copy of Ruby Asciidoctor
+
+The transpilation relies on a copy of the Ruby source in `packages/core/build/asciidoctor`.
+The build instructions above supply this by unpacking a tar.gz of the repository constructed by GitHub.
+Alternatively, you can supply this copy yourself.
+For the following posix-centric suggestions, assume `ASCIIDOCTOR_RB` is the local location of your Asciidoctor Ruby project.
+
+* Remove the current (contents of the) Ruby source directory: `rm -rf packages/core/build/asciidoctor`
+* Choose a method:
+** Using a symlink: `ln -s $ASCIIDOCTOR_RB packages/core/build/asciidoctor`
+** Copying: `cp -r $ASCIIDOCTOR_RB packages/core/build/`
+** Unpacking a gem:
+----
```

Comment:
We might use a list continuation to indent the listing block:

```suggestion
+
----
```

Reply:
I liked that so much I changed most of the other usages!